.github/workflows/e2e_nightly.yml

```diff
@@ -111,7 +119,7 @@ jobs:
         -GNinja
         -B ${{github.workspace}}/build-e2e/
         -S ${{github.workspace}}/sycl-repo/sycl/test-e2e/
-        -DSYCL_TEST_E2E_TARGETS="ext_oneapi_cuda:gpu"
+        -DSYCL_TEST_E2E_TARGETS="${{matrix.adapter.prefix}}_${{matrix.adapter.str_name}}:${{matrix.adapter.unit}}"
```

Comment:
this `_` between prefix and name seems not right 😉

Reply:
This is intentional. It is part of the target name.

Reply:
What I meant is - it won't work in case of an empty prefix...?

Reply:
Fixed.